packages/sync/src/Sender.php

```diff
@@ -512,6 +513,38 @@ public function do_sync_for_queue( $queue ) {
 		return true;
 	}
 
+	/**
+	 * Returns any object that is able to be synced.
+	 *
+	 * @access public
+	 *
+	 * @param array $args the synchronized object parameters.
+	 * @return string Encoded sync object.
+	 */
+	public function sync_object( $args ) {
+		// For example: posts, post, 5.
+		list( $module_name, $object_type, $id ) = $args;
+
+		$sync_module = Modules::get_module( $module_name );
```

Comment:
I believe we would need to move the `use` statement to this file too:
```
use Automattic\Jetpack\Sync\Modules;
```

Reply:
I see why you suggested that, but actually it is not necessary, because `Modules` and `Sender` are in the same namespace, and in that case no `use` statement is necessary.